### DISCUSSION TITLE
dashboard not loading

### DISCUSSION THREAD

**Discussion by Tobi7241** [beep-projects/SystaPi#20] [General]
any idea ??

http://systapi:1337/SystaREST/dashboard does not load and only show 
"../../helpers/systapidashboard_flat.html"

/home/beep/helpers/ has this file:
  -rwxr-xr-x 1 beep beep 786952 19. Dez 17:20 systapidashboard_flat.html

Server is up, status page loads with correct info and shows 60 logged entries:
http://systapi:1337/SystaREST/servicestatus
{
"timeStampString": "2023-12-20T18:19:33.85417+01:00",
"connected": false,
"running": true,
"lastDataReceivedAt": "2023-12-20T18:19:24.55+01:00",
"packetsReceived": 80,
"paradigmaListenerIP": "192.168.1.1",
"paradigmaListenerPort": 22460,
"paradigmaIP": "192.168.1.13",
"paradigmaPort": 8002,
"loggingData": false,
"logFileSize": 60,
"logFilePrefix": "DataLogger",
"logFileDelimiter": ";",
"logFileRootPath": "/home/beep/logs",
"logFilesWritten": 0,
"logBufferedEntries": 60,
"commitDate": "2023-10-16T16:25:20+00:00"
}

**Comment by beep-projects** [comment]
Did you run the build.sh script?
you need to copy the file into the bin directory and rename it
`cp ~/helpers/systapidashboard_flat.html ~/SystaRESTServer/bin/systapidashboard.html`

**Reply by Tobi7241** [reply]
indeed. copying the files manually resolved that issue. I did run build.sh at some point, yes.
